### ISSUE TITLE
Python docs are missing in-between version updates

### ISSUE THREAD

**Issue by monk-time** [Thibaut/devdocs#458]
I stumbled upon this issue by being unable to find documentation for `typing.NewType` [introduced in Python 3.5.2](https://docs.python.org/3/whatsnew/changelog.html#python-3-5-2). Comparing [the page for the `typing` module on DevDocs](https://devdocs.io/python%7E3.5/library/typing#type-aliases) with [its official source](https://docs.python.org/3.5/library/typing.html#type-aliases), you can see that the first section should have more content, and the section following it is missing.

Looking further, I have noticed that [the official docs](https://docs.python.org/3.5/library/index.html) for the latest stable Python version (with "Python » 3.5.2 Documentation" showing on the top) keep receiving updates even after the release on 2016-06-26. [According to their repo](https://github.com/python/cpython/commits/3.5/Doc/library?page=2), there are about 60 commits on their 3.5 branch touching the docs as shown on their website that are missing in the DevDocs snapshot. "Issue #24136" from Jun 12 is there (listed in the changelog under "Python 3.5.2 release candidate 1"), but not the following commit "Issue #27221", or "Issue #16182" (listed under "Python 3.5.3 release candidate 1").

My guess is that the missing changes are technically meant to 'ship' with 3.5.3. But since they seem to be mostly fixes for mistakes or missing/outdated info in the docs, they are shown on the official site immediately, even the ones commited as recently as today ([[1]](https://docs.python.org/3.5/library/typing.html#typing.AnyStr), [[2]](https://github.com/python/cpython/commit/2a50a4e347bdf898e629d3078d521295072e3977)).

In other words, if that's not too much effort, I propose that the Python docs should be (auto-)updated more often.

**Comment by Thibaut** [comment]
Updating docs is a manual process. I can only do it as often as my spare time allows and must balance spending time updating docs with adding new ones and improving the app. Updating docs in between releases doesn't make the cut because most of the time there are no significant changes in the documentation, and I'd rather spend my time on higher-impact things.

Fully automating the update process is not realistic at this stage given the architecture of DevDocs and my limited spare time.

**Comment by monk-time** [comment]
I understand, and it's totally fine. I was just surprised to discover that Python docs receive these updates continuously, and thought it was accidentally not taken into account in DevDocs.
